internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -83,6 +83,12 @@ func resourceCdnFrontDoorFirewallPolicy() *pluginsdk.Resource {
 				ValidateFunc: validation.IsURLWithScheme([]string{"http", "https"}),
 			},
 
+			"request_body_check": {
```

Comment:
as this is a boolean the convention in the provider is to put enabled into the name such as
```suggestion
			"request_body_check_enabled": {
```